content/en/flux/installation/_index.md

```diff
@@ -0,0 +1,308 @@
+---
+title: "Installation"
+description: "Flux install, bootstrap, upgrade and uninstall documentation."
+weight: 30
+---
+
+This guide walks you through setting up Flux to
+manage one or more Kubernetes clusters.
+
+## Prerequisites
+
+You will need a Kubernetes cluster that matches one of the following versions:
+
+| Kubernetes version | Minimum required |
+|--------------------|------------------|
+| `v1.24`            | `>= 1.24.0`      |
+| `v1.25`            | `>= 1.25.0`      |
+| `v1.26`            | `>= 1.26.0`      |
+| `v1.27` and later  | `>= 1.27.1`      |
+
+{{% alert color="info" title="Kubernetes EOL" %}}
+Note that Flux may work on older versions of Kubernetes e.g. 1.19,
+but we don't recommend running [EOL versions](https://endoflife.date/kubernetes)
+in production nor do we offer support for these versions.
+{{% /alert %}}
+
+## Install the Flux CLI
+
+The Flux CLI is available as a binary executable for all major platforms,
+the binaries can be downloaded from GitHub
+[releases page](https://github.com/fluxcd/flux2/releases).
+
+{{< tabpane text=true >}}
+{{% tab header="Homebrew" %}}
+
+With [Homebrew](https://brew.sh) for macOS and Linux:
+
+```sh
+brew install fluxcd/tap/flux
+```
+
+{{% /tab %}}
+{{% tab header="bash" %}}
+
+With [Bash](https://www.gnu.org/software/bash/) for macOS and Linux:
+
+```sh
+curl -s https://fluxcd.io/install.sh | sudo bash
+```
+
+{{% /tab %}}
+{{% tab header="yay" %}}
+
+With [yay](https://github.com/Jguer/yay) (or another [AUR helper](https://wiki.archlinux.org/title/AUR_helpers)) for Arch Linux:
+
+```sh
+yay -S flux-bin
+```
+
+{{% /tab %}}
+{{% tab header="nix" %}}
+
+With [nix-env](https://nixos.org/manual/nix/unstable/command-ref/nix-env.html) for NixOS:
+
+```sh
+nix-env -i fluxcd
+```
+
+{{% /tab %}}
+{{% tab header="Chocolatey" %}}
+
+With [Chocolatey](https://chocolatey.org/) for Windows:
+
+```powershell
+choco install flux
+```
+
+{{% /tab %}}
+{{< /tabpane >}}
+
+To configure your shell to load `flux` [bash completions](./cmd/flux_completion_bash.md) add to your profile:
+
+```sh
+. <(flux completion bash)
+```
+
+[`zsh`](./cmd/flux_completion_zsh.md), [`fish`](./cmd/flux_completion_fish.md),
+and [`powershell`](./cmd/flux_completion_powershell.md)
+are also supported with their own sub-commands.
+
+A container image with `kubectl` and `flux` is available on DockerHub and GitHub:
+
+* `docker.io/fluxcd/flux-cli:<version>`
+* `ghcr.io/fluxcd/flux-cli:<version>`
+
+## Bootstrap
+
+Using the `flux bootstrap` command you can install Flux on a
+Kubernetes cluster and configure it to manage itself from a Git
+repository.
+
```

Comment:
Here we need a short example, you could use the bootstrap git with a private SSH key so we avoid talking about SSH agent here.